kinto_http/batch.py

```diff
@@ -45,18 +50,35 @@ def _build_requests(self):
     def send(self):
         result = []
         requests = self._build_requests()
+        id_request = 0
         for chunk in utils.chunks(requests, self.batch_max_requests):
             kwargs = dict(method='POST',
                           endpoint=self.endpoints.get('batch'),
                           payload={'requests': chunk})
             resp, headers = self.session.request(**kwargs)
             for i, response in enumerate(resp['responses']):
+                logger.debug("\nBatch #{}: \n\tRequest: {}\n\tResponse: {}\n".format(
+                    id_request, json.dumps(chunk[i]), json.dumps(response)))
                 status_code = response['status']
                 if not (200 <= status_code < 400):
                     message = '{0} - {1}'.format(status_code, response['body'])
                     exception = KintoException(message)
                     exception.request = chunk[i]
                     exception.response = response
+                if status_code > 499:
                     raise exception
+                else:
+                    if status_code < 400:
+                        log = logger.warn
+                        message = response["body"].get("message", "")
+                    else:
+                        log = logger.error
+                        message = response["body"]["message"]
+                    log("Batch #{}: {} {} - {} {}".format(
+                        id_request, chunk[i]["method"], chunk[i]["path"],
+                        status_code, message))
```

Comment:
I think it would be cleaner to do it like this:

```py
level = logging.WARN if status_code < 400 else logging.ERROR
message = response["body"].get("message", "")
logger.log(level, "Batch....")
```